velox/connectors/hive/HiveConfig.h

```diff
@@ -107,6 +107,17 @@ class HiveConfig {
   /// Maximum number of entries in the file handle cache.
   static constexpr const char* kNumCacheFileHandles = "num_cached_file_handles";
 
+  /// Maximum stripe size for orc file.
+  static constexpr const char* kOrcWriterMaxStripeSize =
+      "orc_optimized_writer_max_stripe_size";
+  static constexpr const char* kOrcWriterMaxStripeSizeConfig =
+      "hive.orc.writer.stripe-max-size";
+
+  static constexpr const char* kOrcWriterMaxDictionaryMemory =
```

Comment:
Why we need to define a pair of configs but not just one as "hive.orc.writer.dictionary-max-memory"? Thanks!